### DISCUSSION TITLE
Setting default model

### DISCUSSION THREAD

**Discussion by brknsoul** [Bing-su/adetailer#76] [Q&A] [answered]
Is there a config file I can edit where I can set the default model, rather than having to change it each time?

**Comment by Bing-su** [comment]
Learn how to modify the `ui-config.json` file...

**Comment by brknsoul** [comment, marked as answer]
Semi-self-answered: Open the `ui-config.json` and look for `"customscript/!adetailer.py/txt2img/ADetailer model/value"` and modify the value there.
